mysql-compatibility.md

```diff
@@ -1,17 +1,15 @@
 ---
 title: Compatibility with MySQL
-summary: Learn about the compatibility of TiDB with MySQL, and the unsupported and different features.
+summary: Compares TiDB and MySQL in syntax and features.
 category: reference
 aliases: ['/docs/dev/reference/mysql-compatibility/']
 ---
 
 # Compatibility with MySQL
 
-TiDB supports both the MySQL wire protocol and the majority of its syntax. This means that you can use your existing MySQL connectors and clients, and your existing applications can often be migrated to TiDB without changing any application code.
+TiDB is fully compatible with the MySQL 5.7 protocol and the common features and syntax of MySQL 5.7. The ecosystem tools for MySQL 5.7 (PHPMyAdmin, Navicat, MySQL Workbench, mysqldump, and Mydumper/myloader) and the MySQL client can be used for TiDB.
 
-Currently TiDB Server advertises itself as MySQL 5.7 and works with most MySQL database tools such as PHPMyAdmin, Navicat, MySQL Workbench, mysqldump, and mydumper/myloader.
-
-However, TiDB does not support some of MySQL features or behaves differently from MySQL because these features cannot be easily implemented in a distributed system. For some MySQL syntax, TiDB can parse but does not process it. For example, the `ENGINE` table option in the `CREATE TABLE` statement can be parsed but is ignored.
+TiDB is a distributed database. However, some features have not yet been implemented in TiDB or are only syntactically supported because these features are difficult for engineering or expected to have low output compared with the input.
```

Comment:
@tangenta PTAL
```suggestion
However, some features of MySQL have not been implemented in the distributed database TiDB yet, or are only syntactically supported, because these features are difficult to implement or have low ROI (Return On Investment).
```